### PR TITLE
✨ feat: miscellanous improvements

### PR DESCRIPTION
**Pull request by pallabpain**
### Description

This pull request addresses multiple requests. Please check the commits for more details.

### CODE REVIEW

File: riocli/apply/__init__.py [hunk 1]
```diff
@@ -35,11 +35,11 @@
               help='Dry run the yaml files without applying any change')
 @click.option('--show-graph', '-g', is_flag=True, default=False,
               help='Opens a mermaid.live dependency graph')
-@click.option('--values', '-v',
+@click.option('--values', '-v', multiple=True, default=(),
```
Comment (ankitrgadiya) [inline riocli/apply/__init__.py:38]: todo: We also need this in template command.

File: riocli/apply/__init__.py [hunk 2]
```diff
@@ -35,11 +35,11 @@
...
               help="Path to values yaml file. Key/values "
```
Comment (ankitrgadiya) [inline riocli/apply/__init__.py:39]: suggestion: Can we update help to also mention multiple values are possible? Also, in the docstring example, we can have one multiple value example.

File: riocli/apply/__init__.py (hunk 1 above)
Comment (pallabpain) [reply riocli/apply/__init__.py:38]: Done.

File: riocli/apply/template.py [hunk 3]
```diff
@@ -28,10 +28,10 @@
     help_headers_color=Colors.YELLOW,
     help_options_color=Colors.GREEN,
 )
-@click.option('--values', '-v',
+@click.option('--values', '-v', multiple=True, default=(),
               help='Path to values yaml file. key/values specified in the '
                    'values file can be used as variables in template YAMLs')
-@click.option('--secrets', '-s',
+@click.option('--secrets', '-s', multiple=True, default=(),
```
Comment (ankitrgadiya) [inline riocli/apply/template.py:34]: todo: Please update the type-annotation as well.

**Review by ankitrgadiya — APPROVED** [review]

File: riocli/device/model.py [hunk 4]
```diff
@@ -79,6 +80,9 @@ def apply(self, *args, **kwargs) -> ApplyResult:
         onboard_command = onboard_script.full_command()
         execute_onboard_command(hwil_response.id, onboard_command)
 
+        if self.spec.get('virtual', {}).get('wait', False):
```
Comment (ankitrgadiya) [inline riocli/device/model.py:83]: suggestion(non-blocking): Maybe we can extract the `virtual` object before line 40.

Comment (pallabpain) [reply riocli/device/model.py:83]: Done.

**Pull request merged**